hbw/app/javascript/packs/hbw/components/buttons.js.jsx

```diff
@@ -69,19 +39,44 @@ modulejs.define(
         }
       });
 
+      requestConnection = () => {
+        const data = {
+          entity_code:  this.props.entityCode,
+          entity_type:  this.props.entityTypeCode,
+          entity_class: this.props.entityClassCode
```

Comment:
I recommend prefetch all props on start of method and then use everywhere you need. I mean
`const { entityCode,  entityTypeCode, entityClassCode, env} = this.props;`
and then
```
const data = {
          entity_code:  entityCode,
          entity_type:  entityTypeCode,
          entity_class: entityClassCode
};
...
errorHeader: env.translator('errors.cannot_obtain_available_actions')
```
It allows to see all the props (or state) used in the method.